src/util/authUtils.js

```diff
@@ -74,14 +76,14 @@ export function validateSecret(secret) {
 }
 
 export const getMemberCookieName = server =>
-	server.app.isDevConfig ? 'MEETUP_MEMBER_DEV' : 'MEETUP_MEMBER';
+	server.app.isDev ? 'MEETUP_MEMBER_DEV' : 'MEETUP_MEMBER';
 
 /**
  * apply default cookie options for auth-related cookies
  */
 export const configureAuthCookies = server => {
-	const password = validateSecret(server.plugins.requestAuth.config.COOKIE_ENCRYPT_SECRET);
-	const isSecure = process.env.NODE_ENV === 'production';
+	const password = validateSecret(config.get('cookie_encrypt_secret'));
```

Comment:
Not sure how this works - is `server.plugins.requestAuth.config` as referenced on line 83 the same as this `config` variable (imported from the config)?